### PR TITLE
fix(condo): DOMA-6135 fixed incident close without "workFinish"

### PR DESCRIPTION
**Pull request by Alllex202**
Problem: it was possible to move an incident to not actual status without a "finshWork" field.

Now it is impossible to do this - an incident with an not actual status must have a "finshWork" field

### CODE REVIEW

File: apps/condo/domains/ticket/schema/Incident.js [hunk 1]
```diff
@@ -27,9 +27,17 @@ const ERRORS = {
     WORK_FINISH_EARLY_THAN_WORK_START: {
         code: BAD_USER_INPUT,
         type: WRONG_VALUE,
+        variable: ['data', 'workFinish'],
         message: 'The value of the "workFinish" field must be greater or equal than the "workStart" field',
         messageForUser: 'api.incident.WORK_FINISH_EARLY_THAN_WORK_START',
     },
+    WORK_FINISH_MUST_BE_SPECIFIED_iF_NOT_ACTUAL_STATUS: {
```
Comment (akarjakin) [inline apps/condo/domains/ticket/schema/Incident.js:34]: iF

File: apps/condo/domains/ticket/schema/Incident.js [hunk 2]
```diff
@@ -27,9 +27,17 @@ const ERRORS = {
     WORK_FINISH_EARLY_THAN_WORK_START: {
```
Comment (akarjakin) [inline apps/condo/domains/ticket/schema/Incident.js:27]: WORK_FINISHED_...

**Review by ekabardinsky — APPROVED** [review]

File: apps/condo/domains/ticket/schema/Incident.js [hunk 3]
```diff
@@ -24,11 +24,19 @@ const {
 
 
 const ERRORS = {
-    WORK_FINISH_EARLY_THAN_WORK_START: {
+    WORK_FINISHED_EARLY_THAN_WORK_START: {
```
Comment (toplenboren) [inline apps/condo/domains/ticket/schema/Incident.js:27]: WORK_FINISHED_EARLIER_THEN_WORK_STARTED ?

File: apps/condo/lang/en/en.json [hunk 4]
```diff
@@ -1574,7 +1574,8 @@
   "api.organization.reInviteOrganizationEmployee.EMPLOYEE_NOT_FOUND": "Could not employee using specified contacts",
   "api.organization.reInviteOrganizationEmployee.ALREADY_ACCEPTED_INVITATION": "Specified employee has already accepted invitation",
   "api.banking.createBankAccountRequest.INCORRECT_PROPERTY_ID": "Incorrect propertyId was provided. Please check that this id related to passed organizationId",
-  "api.incident.WORK_FINISH_EARLY_THAN_WORK_START": "The end of the works cannot be earlier than the start",
+  "api.incident.WORK_FINISHED_EARLY_THAN_WORK_START": "The end of the works cannot be earlier than the start",
```
Comment (toplenboren) [inline apps/condo/lang/en/en.json:1577]:
```suggestion
  "api.incident.WORK_FINISHED_EARLY_THAN_WORK_START": "The end of the work cannot be earlier than the start",
```

**Pull request merged**